packages/mui-material/src/Autocomplete/autocompleteClasses.ts

```diff
@@ -52,6 +52,8 @@ export interface AutocompleteClasses {
   groupLabel: string;
   /** Styles applied to the group's ul elements. */
   groupUl: string;
+  /** Styles applied to the root element if keyboard focused. */
```

Comment:
It is applied to the `li` elements of the Autocomplete options, not the root element.
```suggestion
  /** Styles applied to the option elements if they are keyboard focused. */
```

Reply:
done in this commit https://github.com/mui/material-ui/pull/37502/commits/925569bc44f839b6dbdfe2cacf264d3bc1eaa214